.docforge/documentation/gardener-extensions/others.yaml

```diff
@@ -55,3 +55,11 @@ structure:
       description: Gardener extension controller which deploys pull-through caches for container registries.
     source: https://github.com/gardener/gardener-extension-registry-cache/blob/DEFAULT_BRANCH/README.md
   - fileTree: https://github.com/gardener/gardener-extension-registry-cache/tree/main/docs
+- dir: gardener-extension-shoot-rsyslog-relp
+  structure:
+  - file: _index.md
+    frontmatter:
+      title: Node Audit Logging
+      description: Gardener extension controller which configures the rsyslog an auditd services installed on shoot nodes.
```

Comment:
```suggestion
      description: Gardener extension controller which configures the rsyslog and auditd services in the shoot nodes.
```

Reply:
I copy-pasted from https://github.com/gardener/gardener-extension-shoot-rsyslog-relp/blob/c09737d2ab8780c87463e5a685cc2ed89988a5e8/README.md?plain=1#L6 🙄 

Let me first fix in the README.md of the shoot-rsyslog-relp extension.